spk/tt-rss/src/installer.sh

```diff
@@ -60,12 +60,14 @@ postinst ()
         ${MYSQL} -u root -p"${wizard_mysql_password_root}" -e "CREATE DATABASE ${MYSQL_DATABASE}; GRANT ALL PRIVILEGES ON ${MYSQL_DATABASE}.* TO '${MYSQL_USER}'@'localhost' IDENTIFIED BY '${wizard_mysql_password_ttrss}';"
         ${MYSQL} -u ${MYSQL_USER} -p"${wizard_mysql_password_ttrss}" ${MYSQL_DATABASE} < ${WEB_DIR}/${PACKAGE}/schema/ttrss_schema_mysql.sql
         single_user_mode=$([ "${wizard_single_user}" == "true" ] && echo "true" || echo "false")
-        sed -e "s|define('DB_TYPE', \".*\");|define('DB_TYPE', 'mysql');|" \
-            -e "s|define('DB_USER', \".*\");|define('DB_USER', '${MYSQL_USER}');|" \
-            -e "s|define('DB_NAME', \".*\");|define('DB_NAME', '${MYSQL_DATABASE}');|" \
-            -e "s|define('DB_PASS', \".*\");|define('DB_PASS', '${wizard_mysql_password_ttrss}');|" \
+        sed -e "s|define('DB_TYPE', '.*');|define('DB_TYPE', 'mysql');|" \
+            -e "s|define('DB_HOST', '.*');|define('DB_HOST', 'localhost');|" \
```

Comment:
Basically the configuration introduced a new possibility to configure the db host.